servers/wanaku-server-quarkus/src/main/java/ai/wanaku/server/quarkus/api/v1/tools/ToolsResource.java

```diff
@@ -58,6 +63,7 @@ public Response add(ToolReference resource) {
     @Path("/list")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(responseCode = "200", content = @Content(schema = @Schema(type = SchemaType.ARRAY, implementation = ToolReference.class)))
```

Comment:
In order to be consistent with the resource code, we should add 2 APIResponse, one for 200, and one for 500, ala

```
@Path("/list")
    @GET
    @Produces(MediaType.APPLICATION_JSON)
    @APIResponses({
            @APIResponse(responseCode = "200", content = @Content(
                    schema = @Schema(type = SchemaType.ARRAY, implementation = ResourceReference.class))),
            @APIResponse(responseCode = "500", content = @Content(schema = @Schema(implementation = String.class)))
    })
```
I'll update https://github.com/wanaku-ai/wanaku/pull/110#issuecomment-2698566585

Reply:
Thanks, I added it there